### PR TITLE
Style added book-store-CMS react project

### PR DESCRIPTION
**Pull request by juaryimami**
### In this last project of the Bookstore website:-
---
 - The functionality is done. 
 - I have added styling. 
 - I also used `Zeplin` as a collaboration tool.
 ---
#### Thank your for reviewing my project!!!!.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,30 +3,34 @@
...
     "web-vitals": "^1.1.2"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "craco start",
+    "build": "craco build",
+    "test": "craco test",
```
Comment (DammyShittu) [inline package.json:30]: - [ ] I was unable to start up your project. See the error I get. Kindly fix that. I would recommend you stick to the CRA `react-scripts` commands.

  tyu

- [ ] This is what your app looks like after I change the commands to `react-scripts`. Kindly ensure your styling matches the design exactly after fixing the issues above.

  ryruow

File: README.md [hunk 2]
```diff
@@ -15,12 +15,14 @@ GitHub: [@JUPAC](https://silver-shortbread-797321.netlify.app/)
...
+- Tailwind
 - Gitflow.
 - Linters.
 
 To get a local copy up and running follow these simple steps:
 
-1. Go to the [repository page]( https://juaryimami.github.io/math-magician/).
+1. Go to the [repository page]( https://juaryimami.github.io/book-store-react/).
 2. Press the "Code" button and copy the link.
 3. Clone it using git command `git clone <link>`.
```
Comment (DammyShittu) [inline README.md:28]: - [ ] Kindly update your information here on how to set up your app using `craco` config.

- [ ] Also remember to change `math-magician` here to the correct repo name

  hff

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @juaryimami,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by juaryimami** [comment]
Ohhh sory I was ignored the configuration file that is the problem, it
works fine in my side

Thank you!!!!.

**Review by topeogunleye — APPROVED** [review]
### Hi @juaryimami, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
*Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — COMMENTED** [review]
## Hello @juaryimami 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] I highly recommend you use the [new professional Microverse README template](https://github.com/microverseinc/readme-template) which will document your project more professionally.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**